test/factories.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :year do
+    year { 2000 }
```

Comment:
This is the year that the idea of Stocks in the future was first pitched. 😎

Reply:
Assuming that `Year` is a model, might want to establish a pattern of one factory file per model. That will make it easier to find and maintain these as the project grows. Example from Human Essentials: https://github.com/rubyforgood/human-essentials/tree/main/spec/factories

Reply:
I like the idea of one factory file per model that organization is great as we get bigger!  I don't think it has to hold up this PR but is a good idea for future, what do you think @AbuMareBear?